repo/packages/H/hdfs/3/resource.json

```diff
@@ -0,0 +1,13 @@
+{
+  "images": {
+    "icon-small": "https://downloads.mesosphere.com/hdfs/assets/icon-service-hdfs-small.png",
+    "icon-medium": "https://downloads.mesosphere.com/hdfs/assets/icon-service-hdfs-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/hdfs/assets/icon-service-hdfs-large.png"
+  },
+  "assets": {
+    "uris": {
+      "hdfs-mesos-scheduler": "https://downloads.mesosphere.com/hdfs/hdfs-mesos-0.1.9.tgz",
```

Comment:
I've pushed a change to this branch for this but it doesn't show up on the PR.  let me know if you would like me to create a new branch with a new PR or to have a followup PR.  (closing and reopening a PR doesn't work)